Overview page of miscellaneous coding style rules (#22708)
dotnet/docs: docs/fundamentals/code-analysis/style-rules/miscellaneous-rules.md

---
title: Miscellaneous coding style rules
description: Learn about the various .NET code style rules.
ms.date: 02/06/2021
ms.topic: reference
author: BartoszKlonowski
ms.author: BartoszKlonowski

> dev_langs:
> - CSharp
> - VB

Reviewer: There are no code snippets in the article. This can be deleted.

```suggestion
```

---

# Miscellaneous rules

> Rules that do not belong to other categories regards the code quality.

Reviewer 2: ```suggestion
This section contains code-style rules that don't fit in any other category. The miscellaneous rules are:
```

> The rules in this section concern the following miscellaneous rules:

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```
